scout/server/blueprints/variants/templates/variants/cancer-sv-variants.html

```diff
@@ -175,8 +175,8 @@
         </a>
       </div>
       <div class="col-10 d-flex">
-        {{ form.filter_variants(class_="btn btn-primary form-control") }}
-        {{ form.clinical_filter(class_="btn btn-info form-control") }}
+        {{ form.filter_variants(class_="btn btn-primary form-control", onclick="resetPage()") }}
```

Comment:
Oooh, this is an unused macro, isn't it? The one that is called is as the others in utils. Perhaps a good time to merge them? Or to make an issue, its hardly a rush.

Reply:
Let's just remove it, no?